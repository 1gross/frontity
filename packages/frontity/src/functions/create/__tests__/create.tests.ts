describe("create", () => {
  test("goes throw all steps", () => {});
});
