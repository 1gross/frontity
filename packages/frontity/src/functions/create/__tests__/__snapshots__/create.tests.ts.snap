// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create goes throw all steps 1`] = `
Array [
  Array [
    Object {
      "packages": Array [
        "frontity",
        "@frontity/file-settings",
      ],
      "path": "/Users/orballo/Frontity/frontity/packages/frontity",
      "theme": "@frontity/mars-theme",
      "typescript": false,
    },
    Object {
      "name": "random-name",
      "path": "/path/to/project",
    },
  ],
]
`;

exports[`create goes throw all steps 2`] = `
Array [
  Array [
    Object {
      "name": "random-name",
      "path": "/path/to/project",
    },
  ],
]
`;

exports[`create goes throw all steps 3`] = `
Array [
  Array [
    Object {
      "name": "random-name",
      "path": "/path/to/project",
    },
  ],
]
`;

exports[`create goes throw all steps 4`] = `
Array [
  Array [
    "js",
    Object {
      "name": "random-name",
      "path": "/path/to/project",
    },
  ],
]
`;

exports[`create goes throw all steps 5`] = `
Array [
  Array [
    Object {
      "name": "random-name",
      "path": "/path/to/project",
    },
  ],
]
`;

exports[`create goes throw all steps 6`] = `
Array [
  Array [
    Object {
      "name": "random-name",
      "path": "/path/to/project",
    },
  ],
]
`;

exports[`create uses the emitter passed in options to log messages 1`] = `
Array [
  Array [
    "create",
    "Ensuring [33m/path/to/project[39m directory.",
    undefined,
  ],
  Array [
    "create",
    "Creating [33mpackage.json[39m.",
    undefined,
  ],
  Array [
    "create",
    "Creating [33mfrontity.settings.js[39m.",
    undefined,
  ],
  Array [
    "create",
    "Cloning [32mundefined[39m.",
    undefined,
  ],
  Array [
    "create",
    "Installing dependencies.",
    undefined,
  ],
  Array [
    "create",
    "[1m[22m
[1mFrontity project created.[22m

You can start development with [1m[32mfrontity dev[39m[22m.
For documentation visit [4m[35mhttps://docs.frontity.org/[39m[24m.
",
    undefined,
  ],
]
`;

exports[`create uses the emitter passed in options to log the errors 1`] = `
Array [
  Array [
    "create",
    "[1m[31mError: [39m[22m[31mMocked Error[39m
",
    undefined,
  ],
]
`;
